praw/const.py

```diff
@@ -56,6 +56,7 @@
     'list_wikicontributor':   'r/{subreddit}/about/wikicontributors/',
     'live_contributors':      'live/{id}/contributors',
     'live_invite':            'live/{id}/invite_contributor',
+    'live_leave':             'api/live/{id}/leave_contributor',
```

Comment:
The prefix `api` is [optional for some endpoints](https://github.com/reddit/reddit-plugin-liveupdate/blob/a7cde1cf917c4118c4cb114c73a92f272e439816/reddit_liveupdate/__init__.py#L176-L180). 

(Since it's undocumented, `live/{id}/invite_contributor' and 'live/{id}/rm_contributor_invite',  which I wrote, should be replaced with `api`-prefixed version. Sorry.)

> Actually, according the the docs, all the other paths should have api in them as well.

In the docs, some endpoints starts with `/live`:

* GET /live/:thread
* GET /live/:thread/about
* GET /live/:thread/contributors
* GET /live/:thread/discussions

For /live/:thread/about, /api/live/:thread/about [is also used in the doc](https://www.reddit.com/dev/api#GET_api_live_happening_now):

> See also: /api/live/thread/about.